btr-web/btr-common-components/app.config.ts

```diff
@@ -5,9 +5,8 @@ export default defineAppConfig({
     red: 'red',
     button: {
       variant: {
+        ghost: 'hover:bg-inherit',
```

Comment:
Just a comment to ensure when changing base ui variant styles to verify they still look right where they are used across the app as ghost is a commonly used variant

Reply:
yep, I did go through and verify where 'ghost' is used